wiki/compatibility.md

```diff
@@ -0,0 +1,8 @@
+## Compatibility
+
+* Maven 3.6.1
```

Comment:
and higher versions?

Reply:
In pom.xml maven dependency mentioned as 3.6.1. Hence mentioned it as 3.6.1 as it is tested with this version.